### DISCUSSION TITLE
home button

### DISCUSSION THREAD

**Discussion by SiriusXT** [TriliumNext/Trilium#3535] [Show and tell]
![图片](https://user-images.githubusercontent.com/37627919/212825839-c072a2e0-f13d-4847-bba9-f6fc7a70e1e6.png)
`api.addButtonToToolbar({
    title: 'Home',
    icon: 'home-alt-2',
    action: async () => {
       api.activateNote("root");
    }
});
`

**Comment by trailjeep** [comment]
I have mine set to collapse also.
```
api.addButtonToToolbar({
    title: 'Home',
    icon: 'home',
    shortcut: 'alt+h',
    action: async function() {
        api.activateNote('root');
        api.triggerEvent("collapseTree");
    }
});
```

**Reply by Kani3D** [reply]
I'm pretty new to triliumnotes but immediately looked for a homebutton... how can i implement this?

**Reply by trailjeep** [reply]
Actually, now it's easy.

1. right-click launchbar  -> Configure Launchbar
2. right-click Visible Launchers -> Add a note launcher
   2a. change note title to "Home"
   2b. change note icon to home symbol
   2a. target = HOMEPAGE
   2b. keyboardShortcut = alt+h
3. drag & drop to launchbar position

**Reply by Kani3D** [reply]
Ahhh. Thank you so much.
I was not aware of that.
Never thought about right-clicking and adding notes there.
Spread the word.
